spec/api/datastore/maat/maat_ready_application_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe 'get maat ready application' do
+  describe 'GET /api/maat/applications/:usn' do
+    let(:api_request) { get "/api/maat/applications/#{application_usn}" }
+
+    let(:application_usn) { application.application['reference'] }
```

Comment:
Can we put the `application_usn` below the `application` declaration. I know that the file is parsed completely so that `application.application['reference']` is in scope at this point. But I think it is easier to parse as a human if you see `application` first, then the call to it to get the reference :)

Reply:
Yeah that's a good point!